exercises/sending-signals-client/solution/signalclient.py

```diff
@@ -0,0 +1,14 @@
+import asyncio
+import time
+
+from temporalio.client import Client
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_workflow_handle("signals")
+    await handle.signal("submit_greeting", "User 1")
+    time.sleep(1)
```

Comment:
Do not block thread in async call, use `await asyncio.sleep(1)` if this is needed